docs/feature_split_keyboard.md

```diff
@@ -79,6 +79,17 @@ If you're using a custom transport (communication method), then you will also ne
 SPLIT_TRANSPORT = custom
 ```
 
+### Layout Macro
+When setting `SPLIT_KEYBOARD = yes`, row pins are automatically doubled.
+This is because QMK views a split keyboard as follows, with the numbers
+representing row, column:
+
+IMG1
+
+IMG2
```

Comment:
These aren't valid markdown images. Also, please beautify the images you're proposing a bit.
eg. crop/center/remove unnecessary borders/etc.

Reply:
Yes, I didn't know how to go about uploading the image.

I also didn't create this images, @fauxpark did, and might be able to fulfill your request.

Reply:
Imgur would be the preferred way to host them. The changes can be done by you, but here
[https://i.imgur.com/2bYyl07.png](https://i.imgur.com/2bYyl07.png)
[https://i.imgur.com/gGyCL1C.png](https://i.imgur.com/gGyCL1C.png)

Reply:
Sorry, I misunderstood the request. Appreciate you doing that!

Reply:
@fauxpark provided new images and text below since my last visit. Marking as resolved.